tools/bundle_adjust_tracks.cxx

```diff
@@ -1018,6 +1021,22 @@ static int maptk_main(int argc, char const* argv[])
   //
   // Write the output KRTD files
   //
+  if (config->get_value<bool>("krtd_clean_up") )
+  {
+    std::cout << "Cleaning " << config->get_value<std::string>("output_krtd_dir") << " before writing new files." << std::endl;
```

Comment:
Other status messages are going to std::cerr, this should probably be the same for consistency.

This is temporary.  In a future branch (pending release approval) these will all change to use the vital::logger anyway.